### PR TITLE
[NFC][SYCLomatic] Rename APINames*.inc files

### PR DESCRIPTION
**Pull request by yihwang-nv**
Signed-off-by: Wang, Yihan <yihan.wang@intel.com>

### CODE REVIEW

**Review by zhimingwang36 — CHANGES_REQUESTED** [review]
The 1st line of the *inc files contains old filename should be updated. 

E.g. :  //===--------------- APINamesCooperativeGroups.inc ------------------------===// 
the APINamesCooperativeGroups.inc should be updated with the new filename.

**Comment by yihwang-nv** [comment]
> The 1st line of the *inc files contains old filename should be updated.
> 
> E.g. : //===--------------- APINamesCooperativeGroups.inc ------------------------===// the APINamesCooperativeGroups.inc should be updated with the new filename.

done

File: clang/lib/DPCT/Rewriters/CUFFT/CallExprRewriterCUFFT.cpp [hunk 1]
```diff
@@ -1,13 +1,13 @@
-//===--------------- CallExprRewriterCUFFT.cpp ----------------------------===//
+//===--------------- .cpp ----------------------------===//
```
Comment (yihwang-nv) [inline clang/lib/DPCT/Rewriters/CUFFT/CallExprRewriterCUFFT.cpp:1]: //===--------------- CallExprRewriterCUFFT.cpp ----------------------------===//

Comment (yihwang-nv) [reply clang/lib/DPCT/Rewriters/CUFFT/CallExprRewriterCUFFT.cpp:1]: done

File: clang/lib/DPCT/Rewriters/CooperativeGroups/CallExprRewriterCG.inc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//===--------------- APINamesCooperativeGroups.inc ------------------------===//
+//===--------------- CallExprRewriterCG.inc.inc ---------------------------===//
```
Comment (AndyCHHuang) [inline clang/lib/DPCT/Rewriters/CooperativeGroups/CallExprRewriterCG.inc:1]: deplicate inc

Comment (yihwang-nv) [reply clang/lib/DPCT/Rewriters/CooperativeGroups/CallExprRewriterCG.inc:1]: done

**Review by AndyCHHuang — APPROVED** [review]
LGTM

**Pull request closed without merging**